src/Common/Thumbnail/Thumbnail.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import cx from "classnames";
+import { Link } from "react-router-dom";
+
+import "./Thumbnail.scss";
+
+function Thumbnail({ img, title, subtitle, className, to }) {
+  let titleElement = null;
+  let subtitleElement = null;
+
+  if (title) {
+    titleElement = <h1 className="title">{title}</h1>;
+  }
+
+  if (subtitle) {
+    subtitleElement = <h4 className="subtitle">{subtitle}</h4>;
+  }
+
+  return (
+    <Link to={to} className={cx("thumbnail-container", className)}>
+      <div className="background" style={{ backgroundImage: `url(${img})` }} />
+
+      {titleElement}
+      {subtitleElement}
```

Comment:
Just out of curiosity, why don't these two sit within the div with the background image?

Reply:
Because on hover, the contrast/brightness/colour of that div changes, and I don't want those changes to affect the text itself.